### PR TITLE
Do classes exercise

### PR DESCRIPTION
**Pull request by CBinh710**
# Do classes exercise
### Task: [link](https://github.com/users/CBinh710/projects/8?pane=issue&itemId=80465329)
image

### CODE REVIEW

File: src/exercises/class.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    this.name = name;
+    this.age = age;
+  }
+
+  getInfo() {
+    return `This is ${this.name}. He is ${this.age} years old.`;
+  }
+
+  isAdult() {
+    return this.age >= 18;
```
Comment (phuocduong-agilityio) [inline src/exercises/class.js]: Pls review this code

Comment (CBinh710) [reply src/exercises/class.js]: I fixed it, thanks

Comment (phuocduong-agilityio) [reply src/exercises/class.js]: Not work for me. In this code, you got a mistake that I commented before

Comment (CBinh710) [reply src/exercises/class.js]: I put variable for age 18 , thanks for your contribution

File: src/exercises/class.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    this.age = age;
+  }
+
+  getInfo() {
+    return `This is ${this.name}. He is ${this.age} years old.`;
+  }
+
+  isAdult(adultAge) {
+    adultAge = 18;
+    return this.age >= adultAge ? true : false;
```
Comment (phuocduong-agilityio) [inline src/exercises/class.js]: Pls review ur code again, all your function is no correct for some point:
- Your function defination
- Code style is not correct

File: src/exercises/class.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    this.name = name;
+    this.age = age;
+  }
+
+  getInfo() {
+    return `This is ${this.name}. He is ${this.age} years old.`;
+  }
+
+  isAdult(adultAge = 18) {
+    return this.age >= adultAge;
```
Comment (phuocduong-agilityio) [inline src/exercises/class.js:32]: Why we need adultAge as param? It mean adult age can change?

Comment (CBinh710) [reply src/exercises/class.js:32]: In this case adult age can not be changed, so i will put `const adultAge = 18;` to define it

**Pull request merged**